diff 845f8d12e4e417aaafaa6e62a80bf03856e7d906 uncommitted
--- a/sys/src/cmd/faces/main.c
+++ b/sys/src/cmd/faces/main.c
@@ -73,7 +73,7 @@
 ulong	now;
 
 Point	datep = { 8, 6 };
-Point	facep = { 8, 6+0+4 };	/* 0 updated to datefont->height in init() */
+Point	facep = { 8, 6 };	/* 0 updated to datefont->height in init() */
 Point	enddate;			/* where date ends on display; used to place arrows */
 Rectangle	leftr;			/* location of left arrow on display */
 Rectangle	rightr;		/* location of right arrow on display */
@@ -98,7 +98,7 @@
 	initplumb();
 
 	/* make background color */
-	bgrnd = allocimagemix(display, DPalebluegreen, DWhite);
+	bgrnd = allocimage(display, Rect(0,0,1,1), screen->chan, 1, 0xE2E3FAFF);
 	blue = allocimage(display, Rect(0,0,1,1), screen->chan, 1, 0x008888FF);	/* blue-green */
 	left = allocimage(display, leftright, GREY1, 0, DWhite);
 	right = allocimage(display, leftright, GREY1, 0, DWhite);
@@ -119,9 +119,9 @@
  		mediumfont = font;
 	datefont = font;
 
-	facep.y += datefont->height;
+	facep.y += 0;
 	if(datefont->height & 1)	/* stipple parity */
-		facep.y++;
+		facep.y += 0;
 	faces = nil;
 }
 
@@ -351,7 +351,7 @@
 	free(ofaces);
 	nfaces++;
 	setlast();
-	drawarrows();
+	/* drawarrows(); */
 	faces[0] = f;
 	drawface(f, 0);
 	flushimage(display, 1);
@@ -430,7 +430,7 @@
 	memmove(faces+j, faces+j+1, (nfaces-(j+1))*sizeof(Face*));
 	nfaces--;
 	setlast();
-	drawarrows();
+	/* drawarrows(); */
 }
 
 void
@@ -493,7 +493,7 @@
 	drawtime();
 	for(i=0; i<nfaces; i++)
 		drawface(faces[i], i);
-	drawarrows();
+	/* drawarrows(); */
 	flushimage(display, 1);
 }
 
@@ -712,11 +712,11 @@
 	init();
 	unlockdisplay(display);	/* initdraw leaves it locked */
 	display->locking = 1;	/* tell library we're using the display lock */
-	setdate();
+	/* setdate(); */
 	eresized(0);
 
 	pids[Mainp] = getpid();
-	startproc(timeproc, Timep);
+	/* startproc(timeproc, Timep); */
 	startproc(mouseproc, Mousep);
 	if(initload)
 		for(i = 0; i < nmaildirs; i++)
